### PR TITLE
Updating Getting Started with Resources section #68962

### LINKED ISSUE #68962: Include demo/example repo options in getting  started page
##### SUMMARY

Ansible provides a couple of good resources for learning/experimenting with Ansible. Add these to getting started:

 https://github.com/ansible/product-demos
https://katacoda.com/rhel-labs 

##### ISSUE TYPE
- Documentation Report

##### COMPONENT NAME

docs.ansible.com
##### ANSIBLE VERSION

```paste below

```

##### CONFIGURATION

```paste below

```

### PR DESCRIPTION
**Pull request by kshitijcode**
##### SUMMARY

Updating Getting Started with Resources sectio

##### ISSUE TYPE

- Docs Pull Request

##### ADDITIONAL INFORMATION

```paste below

```
Fixes https://github.com/ansible/ansible/issues/68962

### CODE REVIEW

File: docs/docsite/rst/user_guide/intro_getting_started.rst [hunk 1]
```diff
@@ -109,6 +109,12 @@ You can read more about privilege escalation in :ref:`become`.
 
 Congratulations! You have contacted your nodes using Ansible. You used a basic inventory file and an ad-hoc command to direct Ansible to connect to specific remote nodes, copy a module file there and execute it, and return output. You have a fully working infrastructure.
 
+Resources
+=================================
+
```
Comment (imjoseangel) [inline docs/docsite/rst/user_guide/intro_getting_started.rst]: Remove extra line

File: docs/docsite/rst/user_guide/intro_getting_started.rst [hunk 2]
```diff
@@ -109,6 +109,12 @@ You can read more about privilege escalation in :ref:`become`.
...
+* https://github.com/ansible/product-demos
```
Comment (imjoseangel) [inline docs/docsite/rst/user_guide/intro_getting_started.rst]: Add with URL Format:

 `dictionary.py <https://github.com/imjoseangel/100-days-of-code/blob/devel/python/interactive-dictionary/dictionary.py>`_

```rst
- `Product Demos <https://github.com/ansible/product-demos>`_
- `Katakoda <https://katacoda.com/rhel-labs>`_
```
-  `Product Demos <https://github.com/ansible/product-demos>`_
- `Katakoda <https://katacoda.com/rhel-labs>`_

File: docs/docsite/rst/user_guide/intro_getting_started.rst [hunk 3]
```diff
@@ -109,6 +109,12 @@ You can read more about privilege escalation in :ref:`become`.
...
+* https://katacoda.com/rhel-labs
+
```
Comment (imjoseangel) [inline docs/docsite/rst/user_guide/intro_getting_started.rst:119]: Include workshops and others:

https://github.com/ansible/workshops
https://github.com/ansible/ansible-examples
https://github.com/ansible/ansible-baseline

File: docs/docsite/rst/user_guide/intro_getting_started.rst (hunk 2 above)
Comment (imjoseangel) [reply docs/docsite/rst/user_guide/intro_getting_started.rst]: Sorry @kshitijcode I added formatting for `md` and not for `rst`. Could you please review? Thank you

File: docs/docsite/rst/user_guide/intro_getting_started.rst [hunk 4]
```diff
@@ -109,6 +109,14 @@ You can read more about privilege escalation in :ref:`become`.
 
 Congratulations! You have contacted your nodes using Ansible. You used a basic inventory file and an ad-hoc command to direct Ansible to connect to specific remote nodes, copy a module file there and execute it, and return output. You have a fully working infrastructure.
 
+Resources
+=================================
+* [Product Demos](https://github.com/ansible/product-demos)
+* [Katakoda](https://katacoda.com/rhel-labs)
+* [Workshops](https://github.com/ansible/workshops)
+* [Ansible Examples](https://github.com/ansible/ansible-examples)
+* [Ansible Baseline](https://github.com/ansible/ansible-baseline)
```
Comment (imjoseangel) [inline docs/docsite/rst/user_guide/intro_getting_started.rst]: Could you please change to rst format?

```rst
- `Product Demos <https://github.com/ansible/product-demos>`_
- `Katakoda <https://katacoda.com/rhel-labs>`_
- `Workshops <https://github.com/ansible/workshops>`_
- `Ansible Examples <https://github.com/ansible/ansible-examples>`_
- `Ansible Baseline <https://github.com/ansible/ansible-baseline>`_
```
-  `Product Demos <https://github.com/ansible/product-demos>`_
- `Katakoda <https://katacoda.com/rhel-labs>`_
- `Workshops <https://github.com/ansible/workshops>`_
- `Ansible Examples <https://github.com/ansible/ansible-examples>`_
- `Ansible Baseline <https://github.com/ansible/ansible-baseline>`_

Comment (kshitijcode) [reply docs/docsite/rst/user_guide/intro_getting_started.rst]: @imjoseangel Done 👍

**Comment by imjoseangel** [comment]
To me, the format looks perfect. Besides that, I prefer having a brief explanation in every resource to know what I'm going to find.

**Product Demos** - Repo currently under construction and working on a minimal viable demo for testing purposes.
**Katakoda** - Learn new technologies using real environments directly in your browser
**Red Hat Ansible Automation Platform Workshops** - The Red Hat Ansible Automation Workshops project is intended for effectively demonstrating Ansible's capabilities through instructor-led workshops or self-paced exercises.
**Ansible Examples** - Repository with examples and best practices for building Ansible Playbooks.
**Ansible Baseline** - A baseline playbook for testing Ansible performance

**Comment by kshitijcode** [comment]
@imjoseangel : wouldn't that make it like too much info the same place?

**Comment by imjoseangel** [comment]
> @imjoseangel : wouldn't that make it like too much info the same place?

Yep, I was thinking on give extra info but I think is too much. What I would improve are the `Product Demos` and `Workshops` names. What do you think?

**Comment by kshitijcode** [comment]
I think Product Demos can be "Handy Playbooks"  and I feel Workshops is apt for what the resource repository does.

**Comment by acozine** [comment]
Thanks @kshitijcode and @imjoseangel for the PR, the review, and the great conversation!

**Pull request merged**